examples/documentation_examples/examples.go

```diff
@@ -2956,3 +2957,172 @@ func StableAPIExamples() {
 	StableAPINonStrictExample()
 	StableAPIDeprecationErrorsExample()
 }
+
+func insertSnapshotQueryTestData(mt *mtest.T) {
+	catColl := mt.CreateCollection(mtest.Collection{Name: "cats"}, true)
+	_, err := catColl.InsertMany(context.Background(), []interface{}{
+		bson.D{
+			{"adoptable", false},
+			{"name", "Miyagi"},
+			{"color", "grey-white"},
+			{"age", 14},
+		},
+		bson.D{
+			{"adoptable", true},
+			{"name", "Joyce"},
+			{"color", "black"},
+			{"age", 10},
+		},
+	})
+	require.NoError(mt.T, err)
+
+	dogColl := mt.CreateCollection(mtest.Collection{Name: "dogs"}, true)
+	_, err = dogColl.InsertMany(context.Background(), []interface{}{
+		bson.D{
+			{"adoptable", true},
+			{"name", "Cormac"},
+			{"color", "rust"},
+			{"age", 7},
+		},
+		bson.D{
+			{"adoptable", true},
+			{"name", "Frank"},
+			{"color", "yellow"},
+			{"age", 2},
+		},
+	})
+	require.NoError(mt.T, err)
+
+	salesColl := mt.CreateCollection(mtest.Collection{Name: "sales"}, true)
+	_, err = salesColl.InsertMany(context.Background(), []interface{}{
+		bson.D{
+			{"shoeType", "hiking boot"},
+			{"price", 30.0},
+			{"saleDate", time.Now()},
+		},
+	})
+	require.NoError(mt.T, err)
+}
+
+func snapshotQueryPetExample(mt *mtest.T) error {
+	client := mt.Client
+	db := mt.DB
+
+	// Start Snapshot Query Example 1
+	ctx := context.TODO()
+
+	sess, err := client.StartSession(new(options.SessionOptions).SetSnapshot(true))
```

Comment:
Use the shorthand function `options.Session()` instead of `new(options.SessionOptions)`.

E.g.
```go
sess, err := client.StartSession(options.Session().SetSnapshot(true))
```